src/components/DataDictionaryComponent/DataDictionary/MultiplePDF/PdfTable.js

```diff
@@ -1,38 +1,34 @@
 import React from 'react';
 import { View, StyleSheet } from '@react-pdf/renderer';
-import PdfTableHeader from './PdfTableHeader';
-import PdfTableRow from './PdfTableRow';
-import PdfTitle from './PdfTitle';
+import PdfTitle from '../NodePDF/PdfTitle';
+import NodePdfTable from '../NodePDF/PdfTable';
 
 const styles = StyleSheet.create({
   tableContainer: {
-    display: 'table',
-    width: 'auto',
-    borderStyle: 'solid',
-    borderColor: '#bfbfbf',
-    borderWidth: 1,
-    borderRightWidth: 0,
-    borderBottomWidth: 0,
+    border: '1px solid #C1C1C1',
+    marginTop: '50px',
   },
   spacer: {
     height: '20px',
   },
 });
 
 const PdfTable = ({ data }) => (
-  data.map((elem) => (
+  data.map((node) => (
     <View style={styles.tableContainer}>
       <PdfTitle
-        title={elem.id}
-        nodeClass={elem.class}
-        assignment={elem.assignment}
-        desc={elem.desc}
+        title={node.id}
+        nodeClass={node.class}
+        assignment={node.assignment}
+        desc={node.desc}
+        category={node.category}
       />
-      <PdfTableHeader />
-      <PdfTableRow node={elem} />
+      {/* <PdfTableHeader />
+      <PdfTableRow node={node} /> */}
       <div style={styles.spacer} />
+      <NodePdfTable node={node} />
     </View>
-  ))
-);
+  )
+));
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 0.](https://app.codacy.com/gh/CBIIT/bento-icdc-frontend/pullRequest?prid=8787534)
```suggestion
  ));
```